src/docx/documentRender.test.js

```diff
@@ -0,0 +1,105 @@
+const path = require("path");
+
+const { generateDocument } = require("./generateDocument");
+const { getSpecificData } = require("./specificData");
+const { DocumentData } = require("../utils/utils");
+
+describe("In SPECJALNE decision", () => {
+  const issue = "SPECJALNE";
+  var specificData;
+  beforeEach(() => {
+    specificData = getSpecificData(issue);
+  });
+
+  it("child name should be rendered properly", () => {
+    specificData.child.name = "mock child name";
+    const documentData = new DocumentData(specificData).templateData;
+
+    return generateDocument(issue, documentData)
```

Comment:
Is`return` necessary?

Reply:
See: https://jestjs.io/docs/en/asynchronous#promises